execution-engine/engine-tests/src/support/test_support.rs

```diff
@@ -282,17 +283,21 @@ pub fn create_query_request(
     query_request
 }
 
+#[allow(clippy::too_many_arguments)]
 pub fn create_exec_request(
     address: [u8; 32],
+    payment_file: &str,
+    payment_args: impl contract_ffi::contract_api::argsparser::ArgsParser,
     session_contract_file_name: &str,
+    arguments: impl contract_ffi::contract_api::argsparser::ArgsParser,
```

Comment:
if it's payment_args for payment arguments then this should be
```suggestion
    session_args: impl contract_ffi::contract_api::argsparser::ArgsParser,
```